cfpb/consumerfinance.gov
# Secondary Nav: refactor into new design

## Changes

- Change translations from "In this section" to "Navigate this section"
- Refactor secondary nav to latest designs (see screenshots).


## How to test this PR

1. Visit some pages with a secondary navigation, such as 

http://localhost:8000/data-research/small-business-lending/filing-instructions-guide/2024-guide/
http://localhost:8000/rules-policy/regulations/1002/
http://localhost:8000/paying-for-college/your-financial-path-to-graduation/
http://localhost:8000/enforcement/payments-harmed-consumers/
http://localhost:8000/es/herramientas-del-consumidor/prestamos-para-vehiculos/respuestas/conozca-sus-derechos/

Resize to mobile. These pages should have a new secondary nav appearance (see screenshots).


## Screenshots

Before:
<img width="836" alt="Screenshot 2023-11-07 at 10 10 17 AM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/1e4b175e-76db-4362-8adb-249de85ecbc7">


After:
<img width="833" alt="Screenshot 2023-11-07 at 10 10 01 AM" src="https://github.com/cfpb/consumerfinance.gov/assets/704760/7a0218b6-9f70-4946-a1bd-643067fc3cd2">
